### PR TITLE
Feature/template readmes

### PR DESCRIPTION
**Pull request by nicktaras**

### CODE REVIEW

**Comment by nicktaras** [comment]
Hi @futantan,

Please see this draft PR which includes Readme's for each TokenScript template and Readme at the root of this repository.

The aim of these changes is to help provide the most essential information for developers to get started with included resource links to our TokenScript documentation / related materials per TokenScript.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+## Development
+
+### Prerequisites
+
+- Install [TokenScript CLI](https://www.npmjs.com/package/@tokenscript/cli)
+
+## Docs and Resources
+
+- [Smart Token Launch Pad](https://smart-token-store.vercel.app/)
```
Comment (futantan) [inline README.md]: hi @nicktaras please use our prod env https://launchpad.smartlayer.network/

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+- [Smart Token Launch Pad Docs](https://launchpad-doc.vercel.app/)
+- [Smart Token Layer](https://www.smartlayer.network/)
+- [TokenScript](https://www.tokenscript.org/)
+
+## Device Support
+
+TokenScript enabled technologies:
+
+- [TokenScript Viewer](https://viewer.tokenscript.org/)
+- [TokenScript Launch Pad](https://smart-token-store.vercel.app/)
```
Comment (futantan) [inline README.md]: same here

**Review by futantan — CHANGES_REQUESTED** [review]

File: erc-721/pnpm-lock.yaml
Comment (futantan) [inline erc-721/pnpm-lock.yaml:1]: There is already a `package-lock.json` file, let's remove this one

**Review by futantan — APPROVED** [review]
LGTM

**Pull request merged**